main.css

```diff
@@ -5,72 +5,65 @@
 }
 
 body {
-  margin:0;
-  padding:0;
-  font-family: sans-serif;
-  background: linear-gradient(#041535, #3983d8);
-  height: 100vh;
+    font-family: "Roboto", sans-serif;
+    margin: 0;
 }
 
 .container {
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  width: 400px;
-  padding: 40px;
-  transform: translate(-50%, -50%);
-  background: rgba(30, 43, 68, 0.5);
-  box-sizing: border-box;
-  box-shadow: 0 15px 25px rgba(0,0,0,.6);
-  border-radius: 10px;
-  color: #fff;
+  width: 100%;
+  color: #666;
+  text-align: center;
+  padding: 1em 0;
 }
 
-.container .inputBox {
-  position: relative;
-  padding: 2px;
+#taskForm {
+  width: 60vw;
+  max-width: 40rem;
+  min-width: 300px;
```

Comment:
Are these to ensure height when the screen size changes?

Reply:
@kamara-moses please check this comment

Reply:
@akankshadharkar yes that is to ensure the height when the screen size changes.